app/seeds/questions.py

```diff
@@ -192,11 +192,11 @@ def random_datetime(start_date, end_date):
 def seed_questions():
     for info in user_questions:
         username, questions = info.values()
-        print('username', username)
-        print('questions', questions)
+        # print('username', username)
```

Comment:
delete prints

Reply:
they are deleted with one that is commented out.